ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -57,6 +59,48 @@ objects:
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
 
+    initContainers:
+    - name: share-installer-src
+      image: registry.svc.ci.openshift.org/${NAMESPACE}/pipeline:src
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+        mkdir /tmp/installer-src
+        cp -R . /tmp/installer-src
+
+    - name: bazel-build-smoke
+      image: quay.io/coreos/tectonic-builder:bazel-v0.3
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: artifacts
+        # bazel needs to write to .cache
+        mountPath: /.cache
```

Comment:
Does the cache need to be mounted into the container?  We don't mount it [here][1], and the tarball build seems to work.  After a successful build, you're already [copying the smoke-test binary out of its symlinked location into `/tmp/smoke`][2].  I expect all other Bazel files could die with the container.

[1]: https://github.com/openshift/installer/blob/a47aeb1fcf7ffef94202501fcba30bf46aa02532/hack/test-bazel-build-tarball.sh#L6-L11
[2]: https://github.com/openshift/release/pull/1317/files#diff-4b206e1e0e8bf659dda6283fbd812433R102

Reply:
cleaned up bazel-bin

Reply:
> I kept getting Permission Denied, can't create .cache/bazel-something so creating a writable path for that solved the issue...

Ah, probably because of the random UID Prow uses and the lack of `$HOME`.  More on this issue in #1178 and #1185.  Try something like [this][1] (and if you also go that route for `USER`, you can drop your internal `export` from the script).

[1]: https://github.com/openshift/release/pull/1185/files#diff-bd9e5155029eded47605823ce90faa33R169